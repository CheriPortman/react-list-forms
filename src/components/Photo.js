import React from 'react';

//takes a photo property and displays an image


