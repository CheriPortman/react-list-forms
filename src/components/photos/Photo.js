import React from 'react';
import PropTypes from 'prop-types';
//takes a photo property and displays an image

function Photo({ photo }) {
  return <img src={photo} />;
}

Photo.propTypes = {
  photo: PropTypes.string.isRequired
};

export default Photo;
