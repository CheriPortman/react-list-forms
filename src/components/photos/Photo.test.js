import React from 'react';
import { shallow } from 'enzyme';
import Photo from './Photo';

describe('Photo Componenet', () => {
  it('renders a photo', () => {
    const wrapper = shallow(<Photo photo="../../assets/stitch-on-blue.jpg" />);
    expect(wrapper).toMatchSnapshot();
  });
});
