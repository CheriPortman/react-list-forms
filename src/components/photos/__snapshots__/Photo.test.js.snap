// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Photo Componenet renders a photo 1`] = `
<img
  src="../../assets/stitch-on-blue.jpg"
/>
`;
